src/fu-util.c

```diff
@@ -3045,6 +3046,17 @@ main (int argc, char *argv[])
 		g_setenv ("DISABLE_SSL_STRICT", "1", TRUE);
 	}
 
+	/* this doesn't have to be precise (e.g. using the build-year) as we just
+	 * want to check the clock is not set to the default of 1970-01-01... */
```

Comment:
Won't ssl checks still fail if you have utc vs local or other similar wrong too though?

Reply:
They might, but it's unlikely. The SSL cert duration is ~3 months IIRC and so a few hours here or there isn't going to matter. What I'm trying to catch here is "no CMOS battery and no NTP"

Reply:
Okay